ivy/functional/ivy/data_type.py

```diff
@@ -1619,11 +1619,10 @@ def dtype(
     return current_backend(x).dtype(x, as_native=as_native)
 
 
-@handle_exceptions
-@handle_nestable
-def function_supported_dtypes(fn: Callable, recurse: bool = True) -> Tuple:
+def _function_supported_dtypes(fn: Callable, recurse: bool = True) -> Tuple:
```

Comment:
let's move this helper and `_function_supported_dtypes` at the top of the file following the helpers like `_get_function_list`, etc.

Reply:
done